python-libraries/nanover-ase/src/nanover/ase/openmm/runner.py

```diff
@@ -188,6 +206,26 @@ def reset(self):
     def step(self):
         self.imd.step()
 
+    def load(self, index: int):
+        self._sim_index = int(index % len(self.simulation_list))
```

Comment:
This needs to be documented: indices cycle when using both `load` and `next`.

Reply:
Do you mean in a comment here, a docstring here, or somewhere else? I could also just change it to do whatever the rust server does?